humans/tests.py

```diff
@@ -22,6 +24,26 @@ def create_notification(sent=False, group=None):
                                        sent_at=sent_at,
                                        send_to=group)
 
+def create_expiring_key(days_to_expire):
```

Comment:
You could use the `@with_gpg_obj` decorator here and remove the duplicated code. Nothing wrong with this code, but it would be cleaner.